### ISSUE TITLE
Link for "Collection of separate PDF docs" not working

### ISSUE THREAD

**Issue by thepenguinguy** [NetAppDocs/ontap#1009]
### Page URL

https://docs.netapp.com/us-en/ontap/system-admin/index.html

### Page title

Administration overview with the CLI

### Summary

From any of the ONTAP documentation pages, the link for "Collection of separate PDF docs" under "Available PDFs" does not seem to be working. Tried multiple browsers.

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by thepenguinguy** [comment]
Yes, I am still seeing this. I'm on a Mac using Safari and Firefox.

**Comment by ntap-bmegan** [comment]
@netapp-ehoffman - I'm not sure how it is you were able to _not_ reproduce, as we've already got a dev issue open to fix this globally. (It's a system issue, not an ONTAP issue, and it's not specific to any browser or OS.) If you can demonstrate the feature working correctly, please advise. 

@thepenguinguy - I can't estimate when the fix will be live but I can leave this issue open until then if you'd like to be notified.

**Comment by ntap-bmegan** [comment]
@netapp-ahibbard - The fix is live in production and will be live on your site at the next update.

**Comment by netapp-ahibbard** [comment]
@thepenguinguy - thanks for reporting this. The issue has been resolved.
